src/main/java/services/InstallService.java

```diff
@@ -56,8 +56,8 @@
                     engine.initEnvironment();
                     return new Pair<>(engine, false);
                 } catch (Exception e) {
-                    log.warn("Warning: An error occurred while trying to use the old DB file for app. CommCare will now try to install the new DB file for this app. Error details: Got exception "
-                            + e + " while reinitializing at path " + sqliteDB.getDatabaseFileForDebugPurposes());
+                    log.debug("An error occurred while trying to use the old DB file for app. Error details: Got exception "
+                            + e + " while reinitializing at path " + sqliteDB.getDatabaseFileForDebugPurposes() + " Reinitializing new DB ..");
```

Comment:
do `log.warn` and `log.debug` show up to the user at all?

Reply:
No, this only shows in console. This is a totally normal error and was leading developers to incoorectly believe that there is something wrong with the Formplayer, so making it slightly less intruisive.